src/ServicePulse.Host/app/js/views/monitored-endpoints/monitored-endpoints.controller.js

```diff
@@ -0,0 +1,37 @@
+﻿(function(window, angular, undefined) {
+    'use strict';
+
+    function controller(
+        $scope,
+        $timeout,
+        monitoringService) {
+
+        var timeoutId;
+
+        $scope.$on('$destroy', function () {
+            $timeout.cancel(timeoutId);
+        });
+
+        function updateUI() {
+            monitoringService.getRaw().then(function (data) {
+                $scope.endpoints = data["NServiceBus.Endpoints"];
+
+                timeoutId = $timeout(function () {
```

Comment:
🤢 

Rather than starting off on a timer, I'd rather go with a signalr push after the initial render staright away. The longer we leave a timeout in the less likely it will be that we'll change it. And future whoever-is-going-to-work-on-this is going to curse us for it.

Also, should use $interval instead of $timeout for repeated events that should be triggered.

E.g.
```JavaScript
function updateUI() {
    monitoringService.getRaw().then(function(data) {
        $scope.endpoints = data['NServiceBus.Endpoints'];
    });
}

$interval(updateUI, 5000);
```